src/transport/subscription-worker/subscription-worker.ts

```diff
@@ -1535,7 +1554,9 @@ const publishClientEvent = (client: PubNubClientState, event: SubscriptionWorker
   try {
     receiver.postMessage(event);
     return true;
-  } catch (error) {}
+  } catch (error) {
+    if (client.workerLogVerbosity) console.error(`[SharedWorker] Unable send message using message port: ${error}`);
```

Comment:
nitpicking may be. Is it typo in log message for message port instead message port?

Reply:
@mohitpubnub no, it is the way how I wrote MessagePort type. I don't think that it is that bad in this form :)